src/home_work3/models/UserModel.js

```diff
@@ -0,0 +1,83 @@
+const uuid = require("uuid/v4");
+let Sequelize = require("sequelize");
+let dbConnector = require("../db/dbConnector");
+
+var User = dbConnector.define(
+  "users",
+  {
```

Comment:
Consider breaking this prams object into small pieces to improve readability.

Reply:
Resolved as per the comment